.github/workflows/full.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches:
       - develop
       - main
+    tags:
```

Comment:
Is this needed?

Reply:
Yes. This will trigger the actions on tag pushes. The develop branch builds latest, the main branch builds stable, and tags should build a tagged docker image like "0.4.0" or something.

Reply:
Roger. Resolved.